src/03_modules.py

```diff
@@ -5,27 +5,27 @@
 level operating system functionality.
 """
 
+import os
 import sys
+import platform
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
```

Comment:
you probably want a loop, to print one per line, without it i think it defaults to a list and elements